Product/Production/Adapters/General/CONNECTAdapterPDPOpenSSO/src/main/java/com/sun/identity/xacml/plugins/XSPAXACMLAuthzDecisionQueryHandler.java

```diff
@@ -286,19 +284,18 @@ public XSPAXACMLAuthzDecisionQueryHandler() {
         }
 
         com.sun.identity.saml2.protocol.Response samlpResponse = createSamlpResponse(statement, status.getStatusCode()
-                .getValue());
+            .getValue());
 
         System.out.println("XSPAXACMLAuthzDecisionQueryHandler.handleQuery(), returning " + ":samlResponse=\n"
-                + samlpResponse.toXMLString(true, true));
+            + samlpResponse.toXMLString(true, true));
```

Comment:
Since we are already touching this file, is it possible to remove System.out.println lines and replace them with log statements